specification/playwrighttesting/PlaywrightTesting/main.tsp

```diff
@@ -15,16 +15,19 @@ using Azure.Core.Traits;
 using Autorest;
 
 @server(
-  "https://api.playwright-int.io",
-  "Azure Playwright Service API Endpoint"
+  "https://{region}.api.playwright.microsoft.com",
+  "Microsoft Azure Playwright Service API Endpoint",
+  {
+    region: string,
```

Comment:
A common approach is to define an `endpoint` (instead of the `region` + rest of the endpoint).
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/ContentSafety/main.tsp#L34